filters/ThirdParty/filter_208_MalwareDomains/metadata.json

```diff
@@ -1,7 +1,7 @@
 {
   "filterId": 208,
   "name": "Malware Domains",
-  "description": "malware protection",
+  "description": "locks domains that are known to be used to propagate malware and spyware.",
```

Comment:
Missing a 'B' at the start of description (locks instead of Blocks)

Reply:
ok